src/components/ConversationsSelect.ts

```diff
@@ -0,0 +1,36 @@
+import { joinTextChildren } from './Text';
+import { ConversationsSelect as ConversationsSelectSpec } from '@slack/types';
+import { ContainerProps } from './ContainerProps';
+import { FC } from '..';
+
+export interface ConversationSelectProps extends ContainerProps<string> {
+  initialConversation?: string;
+  actionId?: string;
+  responseUrlEnabled?: boolean;
+}
+
+export const ConversationsSelect: FC<
+  ConversationSelectProps,
+  ConversationsSelectSpec
+> = ({
+  children,
+  initialConversation,
+  actionId,
+  responseUrlEnabled = false,
+}) => {
+  const select: ConversationsSelectSpec = {
+    type: 'conversations_select',
+    initial_conversation: initialConversation,
+    action_id: actionId,
+    response_url_enabled: responseUrlEnabled,
```

Comment:
what does `responseUrl` do?

Reply:
it tells slack to create a one-time webhook for posting a message to the selected channel. it looks like the bot does not even need to be a member of the channel. so this simplifies things significantly when handling the view submission